Now generating patients with Atrial Fibrillation

eedrummer opened the pull request.

eedrummer committed:
Now generating patients with Atrial Fibrillation

cmoesel commented on conditions.go:
```diff
@@ -79,6 +79,13 @@ func GenerateConditions(ctx Context, md []ConditionMetadata) []models.Condition
 	}
 
+	afibChance := rand.Intn(100)
+	// 3% Chance of AFib in the general population according to Chris [citation needed]
```
It's actually 2% of the population under 65, and 9% of the population 65+.  Source: http://www.cdc.gov/dhdsp/data_statistics/fact_sheets/fs_atrial_fibrillation.htm

eedrummer replied:
Depends on how you want to handle this. Currently we only generate people over 65, so I can just change the 3 to 9. More properly, we could add the birthdate to the generator context and check it when generating the condition.

cmoesel replied:
If considering the birth date is fairly trivial (i.e., can be implemented in less than an hour), let's do that.  Otherwise, let's bump it to 9% and add a TODO.

eedrummer committed:
AFib chance now changes based on patient age

Should be noted that we currently don't generate anyone under 65, so the chance
of having AFib will always be 9% per the current demographics generation.

eedrummer committed:
More accurate calculation of whether the person is over 65
